spec/tools/fix_auth/auth_model_spec.rb

```diff
@@ -59,44 +59,11 @@
     end
 
     context "#recrypt" do
-      it "should not upgrade blank column" do
-        subject.fix_passwords(blank, options)
-        expect(blank).not_to be_password_changed
-      end
-
       it "should upgrade old columns" do
+        expect(ManageIQ::Password).to receive(:recrypt).and_return("new")
         subject.fix_passwords(old, options)
         expect(old).to be_password_changed
       end
-
-      it "should not encrypt plaintext columns" do
-        subject.fix_passwords(plain, options)
-        expect(plain).to_not be_password_changed
-      end
-
-      it "should raise exception for bad encryption" do
-        expect { subject.fix_passwords(bad, options) }.to raise_error(ManageIQ::Password::PasswordError)
-      end
-
-      it "should replace for bad encryption" do
-        subject.fix_passwords(bad, options.merge(:invalid => "other"))
-        expect(bad.password).to be_encrypted("other")
-      end
```

Comment:
This was the only one  I expected to go away, cause I've seen this one fail - can't say the same for the others.